update-db.js

```diff
@@ -1,12 +1,19 @@
 var childProcess = require('child_process')
-var pkgUp = require('pkg-up')
+var escalade = require('escalade/sync')
 var path = require('path')
 var fs = require('fs')
 
 var BrowserslistError = require('./error')
 
+// eslint-disable-next-line consistent-return
```

Comment:
I prefer to avoid disabling ESLint. Do we have any other options?

Reply:
The callback signature is `string | falsey` -- the only way to satisfy ESLint will be to return an empty string for false condition. Is that fine with you?

Reply:
> Is that fine with you?

yeap